src/main/java/tech/gruppone/stalker/server/repositories/ConnectionRepository.java

```diff
@@ -15,4 +16,8 @@
   @Query("DELETE FROM Connection WHERE userId = :userId AND organizationId = :organizationId")
   Mono<Void> deleteUserConnection(
       @Param("userId") long userId, @Param("organizationId") long organizationId);
+
+  @Query(
+      "SELECT c.organizationId FROM Connection c, Organization o WHERE c.organizationId = o.id AND c.userId = :id")
```

Comment:
qui non credo serva il join! non funzionava senza?

Reply:
Eh serve comunque in questo caso, avevo fatto delle prove